app/services/apply_annotation_to_paragraphs.rb

```diff
@@ -24,15 +21,9 @@ def initialize(params)
   end
 
   def perform
-    # Adds data-p-idx to paragraph HTML tag 
-    paragraph_node['data-p-idx'] = paragraph_index
-
     if (paragraph_index != start_paragraph && paragraph_index != end_paragraph) || (paragraph_index == end_paragraph && paragraph_index != start_paragraph && end_offset == paragraph_node.text.length)
       # wrap entire p node in annotation
-      paragraph_node.children = annotate_html(paragraph_node.inner_html, handle: false)
-      if kind.in? %w{elide replace}
-        paragraph_node['data-elided-annotation'] = id
```

Comment:
I see this still being used in ResourceSectionWrapper, TheResourceBody, with-annotations/cleanup and with-annotations/h2o/export -- should the references be deleted there too?